cpu_monitor:
  temp_warn: 90.0
  temp_error: 95.0
  usage_warn:  0.90
  usage_error: 1.00
  load1_warn: 0.90
  load5_warn: 0.80
  msr_reader_port: 7634
hdd_monitor:
  hdd_reader_port: 7635
  disks:
  - name: /dev/sda
    temp_warn: 55.0
    temp_error: 70.0
  - name: /dev/nvme0
    temp_warn: 55.0
    temp_error: 70.0
  usage_warn: 0.95
  usage_error: 0.99
mem_monitor:
  usage_warn: 0.95
  usage_error: 0.99
net_monitor:
  devices: [ '*' ]
  usage_warn: 0.95
ntp_monitor:
  server: ntp.nict.jp
  offset_warn: 0.1
  offset_error: 5.0
process_monitor:
  num_of_procs: 5
gpu_monitor:
  temp_warn: 90.0
  temp_error: 95.0
  gpu_usage_warn:  0.90
  gpu_usage_error: 1.00
  memory_usage_warn: 0.95
  memory_usage_error: 0.99
